packages/gatsby-plugin-sharp/src/process-file.js

```diff
@@ -75,13 +76,42 @@ const argsWhitelist = [
 exports.processFile = (file, transforms, options = {}) => {
   let pipeline
   try {
+    // adds gatsby cloud image service to gatsby-sharp
+    // this is an experimental api so it can be removed without any warnings
+    if (process.env.GATSBY_CLOUD_IMAGE_SERVICE_URL) {
```

Comment:
Arguably we should fall back to local processing if this fails.

Reply:
We should fail hard. If this fails something isn't right.

Reply:
Okay!